raiden_contracts/contract_manager.py

```diff
@@ -189,9 +189,12 @@ def get_contracts_deployment_info(
     if module not in DeploymentModule:
         raise ValueError(f'Unknown module {module} given to get_contracts_deployment_info()')
 
+    def module_chosen(to_be_added: DeploymentModule):
```

Comment:
I will approve the PR anyway, just a small type nitpick. Add `-> bool:` return type.

Reply:
I defer this https://github.com/raiden-network/raiden-contracts/issues/811 and try to make a release now.